app/src/main/contrib/components/Responsibilities/Guidance/ResponsibilityGuidanceModelInputsPage.tsx

```diff
@@ -9,30 +9,35 @@ import {settings} from "../../../../shared/Settings";
 import {ContribPage} from "../../../ContribPage";
 import {responsibilityGuidanceModelInputsPageActionCreators} from "../../../actions/pages/responsibilityGuidancePageActionCreators";
 
-import { ResponsibilityGuidancePageProps } from "./ResponsibilityGuidancePageProps";
-import { ResponsibilityGuidanceModelInputsContent2017 } from "./content/ResponsibilityGuidanceModelInputsContent2017";
-import { ResponsibilityGuidanceModelInputsContentLatest } from "./content/ResponsibilityGuidanceModelInputsContentLatest";
-import { ResponsibilityGuidanceTouchstoneNotOpenContent } from "./content/ResponsibilityGuidanceTouchstoneNotOpenContent";
+import {ResponsibilityGuidancePageProps} from "./ResponsibilityGuidancePageProps";
+import {ResponsibilityGuidanceModelInputsContent2017} from "./content/ResponsibilityGuidanceModelInputsContent2017";
+import {ResponsibilityGuidanceModelInputsContentLatest} from "./content/ResponsibilityGuidanceModelInputsContentLatest";
+import {ResponsibilityGuidanceTouchstoneNotOpenContent} from "./content/ResponsibilityGuidanceTouchstoneNotOpenContent";
+import {PageArticle} from "../../../../shared/components/PageWithHeader/PageArticle";
 
 export class ResponsibilityGuidanceModelInputsPageComponent extends React.Component<ResponsibilityGuidancePageProps> {
 
-    render() :JSX.Element {
+    render(): JSX.Element {
 
-        if (!this.props.touchstoneVersion)
-        {
+        if (!this.props.touchstoneVersion) {
             return <LoadingElement/>
         }
 
-        if (this.props.touchstoneVersion.status !== "open")
-        {
-            return <ResponsibilityGuidanceTouchstoneNotOpenContent />
+        if (this.props.touchstoneVersion.status !== "open") {
+            return <ResponsibilityGuidanceTouchstoneNotOpenContent/>
         }
 
         if (settings.is2017Touchstone(this.props.touchstoneVersion.id)) {
-            return <ResponsibilityGuidanceModelInputsContent2017 />
-        }
-        else {
-            return <ResponsibilityGuidanceModelInputsContentLatest />
+            return <ResponsibilityGuidanceModelInputsContent2017/>
+        } else if (settings.is2019Touchstone(this.props.touchstoneVersion.id)) {
+            const inputsPdf = require("./content/guidance-2019-inputs.pdf");
+            return <PageArticle title={"Guidance on model inputs: coverage and demographic data"}>
+                <a href={inputsPdf} target={"_blank"}>
```

Comment:
This page won't actually be linked to for 2019 touchstones, but have added this logic just in case someone visits the url directly (unlikely).